tac/project-updates/2024/2024-Q4-Hyperledger-Caliper.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: 2024 Q4 Hyperledger Caliper
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+
+* 2 active maintainers
+* Questions on Discord and GitHub are periodically answered
+* Contributor discussions happen mostly on Discord
+* The mentorship projects are now complete
+
+# Questions/Issues for the TAC
+
+Not at this time.
+
+# Releases
+
+The latest stable release is v0.6.0; we're also continuously publishing unstable releases after every merged PR (no new releases):
+
+* [npm](https://www.npmjs.com/package/@hyperledger/caliper-cli/v/0.6.0)
+* [DockerHub](https://hub.docker.com/layers/hyperledger/caliper/0.6.0/images/sha256-ace68b58b329a5f2c5bd569cc2f59b7f64e9bbee1f3bb9f4f8a97469b55d7b2b?context=explore)
+
+# Overall Activity in the Past Quarter
+
+[List of merged PRs (27)](https://github.com/hyperledger-caliper/caliper/pulls?q=is%3Apr+is%3Aclosed+merged%3A2024-10-01..2024-12-12)
+
+Summary of activities:
+* New Documentation website
+* Documentation improvements/fixes
+* Removing legacy components
+* Updating dependency versions
+* adding unit tests
+* Bug fixes
+
+# Current Plans
+
+* look at getting all dependencies up to latest levels
+* look at doing another caliper release next year
+
+
+# Maintainer Diversity
+
+[Hyperledger Caliper maintainers](https://github.com/hyperledger-caliper/caliper/blob/08f732a484c93285fdc4df628f10cbc8d454d583/MAINTAINERS.md) in 2024 Q4 (unchanged compared to the previous quarter):
+* Dave Kelsey (UK)
+* Attila Klenik (NTCA|BME, Hungary)
+
+# Contributor Diversity
+
+Contributions from maintainers, the community and the mentorship project mentees.
+
+# Additional Information
+
+[Insights dashboard link](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=caliper&routedFrom=Github&bestPractice=false&repository=all&dateFilters=2024-10-01%20to%202024-12-12&dateRange=2024-10-01%20to%202024-12-12&compare=PP&granularity=week&hideBots=true)
+
+Note, that the output on this panel is not correct. It reports that IBM is still a contributor to Caliper and that is definitely not the case and has not been the case for a long while.
```

Comment:
It does not show IBM at all for me, instead Microsoft.

<img width="801" alt="Screenshot 2025-01-09 at 21 59 50" src="https://github.com/user-attachments/assets/5eb3dca9-376f-43ce-b101-dafdce030f83" />